lesson_9/models/computer.go

```diff
@@ -0,0 +1,24 @@
+package computer
+
+type Computer struct {
+	Cpu string
```

Comment:
words like:
ssh, http, cpu, gpu, ram, ssd, id and so on 
should be appercase, for example: HTTP, GPU etc.
check golang styleguide (-3)